relayer/relay/src/chain/cosmos.rs

```diff
@@ -96,7 +96,7 @@ impl Chain for TendermintChain {
 
 /// Perform a generic `abci_query`, and return the corresponding deserialized response data.
 async fn abci_query(
-    chain: &TendermintChain,
+    chain: &CosmosSDKChain,
     path: TendermintPath,
```

Comment:
Should this still be TendermintPath? Or should it be CosmosSDKPath?

Reply:
It's called TendermintPath because it's the Path struct defined in the Tendermint crate. (Contrary to the Path defined in the IBC modules which is also imported here.)

I can alias it something else, but I don't think it's really CosmosSDKPath.

Reply:
We can call it ABCIPath, maybe that's clearer? (It's from the Tendermint::abci library.)